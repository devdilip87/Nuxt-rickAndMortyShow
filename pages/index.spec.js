import { mount, shallowMount } from '@vue/test-utils'
import Index from './index.vue'

describe('Index', () => {

    test('it generate snapshot test', () => {
        const wrapper = shallowMount(Index)
        expect(wrapper.html()).toMatchSnapshot()
    })
})