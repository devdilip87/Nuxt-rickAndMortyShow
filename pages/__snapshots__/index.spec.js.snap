// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index it generate snapshot test 1`] = `
"<div class=\\"box content\\">
  <app-selected-filter-stub></app-selected-filter-stub>
  <section class=\\"row searchSort\\">
    <app-search-stub></app-search-stub>
    <app-sort-stub></app-sort-stub>
  </section>
  <app-charactor-list-stub></app-charactor-list-stub>
</div>"
`;
